apps/meteor/tests/e2e/delete-account.spec.ts

```diff
@@ -0,0 +1,132 @@
+import { faker } from '@faker-js/faker';
+import type { IUser } from '@rocket.chat/core-typings';
+
+import { DEFAULT_USER_CREDENTIALS } from './config/constants';
+import { AccountProfile, Registration, Utils } from './page-objects';
+import { test, expect } from './utils/test';
+
+test.describe('Delete Own Account', () => {
+	let poAccountProfile: AccountProfile;
+	let poRegistration: Registration;
+	let poUtils: Utils;
+	let userToDelete: IUser & { username: string };
+
+	test.beforeAll(async ({ api }) => {
+		await api.post('/settings/Accounts_AllowDeleteOwnAccount', { value: true });
```

Comment:
Should there be a `200 OK` expect for this request? _(same for the other setting updates)_